svm.cpp

```diff
@@ -2220,9 +2220,11 @@ svm_model *svm_train(const svm_problem *prob, const svm_parameter *param)
 
 			if(svm_one_class_probability(prob,model,prob_density_marks) == 0)
 			{
```

Comment:
please remove { and  } in if/else due to libsvm convention. Thanks

Reply:
Done.